### PR TITLE
Marvin test to verify that adding TCP ports 500,4500 and 1701 in vpn should not fail

### PR DESCRIPTION
**Pull request by sanju1010**
Please refer to JIRA ticket for more details
https://issues.apache.org/jira/browse/CLOUDSTACK-9117

Following is the result info:
Test to add TCP Port Forwarding rule for specific ports(500,1701 and 4500) in VPN ... === TestName: test_08_add_TCP_PF_Rule_In_VPN | Status : SUCCESS ===
ok

---

Ran 1 test in 166.799s

OK

### CODE REVIEW

File: test/integration/component/test_vpn_users.py [hunk 1]
```diff
@@ -451,3 +453,64 @@ def test_07_add_VPN_user_domain_admin(self):
...
+        try:
+            Vpn.create(
+                self.apiclient,
+                publicipid=ip.id,
+                account=self.account.name,
+                domainid=self.account.domainid,
+            )
+        except Exception as e:
+            self.fail("Failed to enable vpn on SourceNAT IP with error: %s" % e)
+
```
Comment (pavanb018) [inline test/integration/component/test_vpn_users.py]: @sanju1010  Can you please add an assert case to confirm that the VPN is actually created ? That will make sure that the UDP ports are actually used by VPN and then same ports can be used by TCP in PF ?

File: test/integration/component/test_vpn_users.py [hunk 2]
```diff
@@ -451,3 +453,64 @@ def test_07_add_VPN_user_domain_admin(self):
...
+            self.services['natrule']['publicport'] = port
+            try:
+                NATRule.create(
+                    self.apiclient,
+                    self.virtual_machine,
+                    self.services["natrule"],
+                    ip.id
+                )
+            except Exception as e:
+                self.fail("Creating PF rule for TCP port %s in VPN failed : %s" % (port, e))
```
Comment (pavanb018) [inline test/integration/component/test_vpn_users.py]: Can you please add assert cases here as well for the PF rules ?

**Comment by rhtyd** [comment]
@sanju1010 can you rebase against latest master and share state of your PR, thanks

**Comment by sanju1010** [comment]
Test to add TCP Port Forwarding rule for specific ports(500,1701 and 4500) in VPN ... === TestName: test_08_add_TCP_PF_Rule_In_VPN | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 574.873s

OK

**Comment by bvbharatk** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 112
 Hypervisor xenserver
 NetworkType Advanced
 Passed=68
 Failed=5
 Skipped=3

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**
* test_vpc_vpn.py

 * ContextSuite context=TestRVPCSite2SiteVpn>:setup Failing since 9 runs

 * ContextSuite context=TestVpcRemoteAccessVpn>:setup Failing since 9 runs

 * ContextSuite context=TestVpcSite2SiteVpn>:setup Failing since 9 runs

* test_volumes.py

 * test_06_download_detached_volume Failed

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failed


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_static_role_account_acls
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_routers.py
test_reset_vm_on_reboot.py
test_snapshots.py
test_deploy_vms_with_varied_deploymentplanners.py
test_login.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

**Comment by karuturi** [comment]
failures are note related. merging

**Pull request merged**